src/executables/util_cs.c

```diff
@@ -4649,9 +4652,71 @@ memmon (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  /* execute phase */
+  error_code = mmon_get_server_info (server_info);
+  if (error_code != NO_ERROR)
+    {
+      goto error_exit;
+    }
+
+  // TODO: mmon_print_server_info (server_info);
+
+  if (module_index >= 0)
```

Comment:
The module_index is initialized to MMON_MODULE_LAST. When the user executes the memon utility with default option, the mmon_get_module_info() is called unexpectedly.

Reply:
Done.